frontend/src/features/projects/common/components/FormikTable.tsx

```diff
@@ -32,7 +38,9 @@ const FormikTable: React.FC<IFormikTableProps> = ({
               name={name}
               columns={columns}
               data={data}
+              onRowClick={onRowClick ?? onRowClick}
               pageCount={1}
+              clickableTooltip={clickableTooltip ?? clickableTooltip}
```

Comment:
does this prevent it from just being an empty tooltip? this looks odd

Reply:
Yeah it's just checking to see if the parent passed down a prop for `clickableTooltip`. Essentially the same as `clickableTooltip ? clickableTooltip : null`